SQift.podspec

```diff
@@ -1,14 +1,14 @@
 Pod::Spec.new do |s|
   s.name = "SQift"
-  s.version = "4.0.0"
+  s.version = "4.1.0"
   s.license = "MIT"
   s.summary = "A lightweight Swift wrapper for SQLite."
   s.homepage = "https://github.com/Nike-Inc/SQift"
   s.authors = { "Dave Camp" => "dave.camp@nike.com", "Christian Noon" => "christian.noon@nike.com" }
 
   s.source = { :git => "https://github.com/Nike-Inc/SQift.git", :tag => s.version }
   s.source_files = "Source/**/*.swift"
-  s.swift_version = "4.2"
+  s.swift_versions = ['4.2', '5.0']
```

Comment:
Could we swap single quotes out for double quotes?

Reply:
Done